core/src/main/scala/com/mesosphere/usi/core/Scheduler.scala

```diff
@@ -43,9 +44,14 @@ import scala.concurrent.Future
  *                                     +----------------------+
  */
 object Scheduler {
+
+  private val log = Logger.takingImplicit[KvArgs](getClass)
+
   case class MesosConnection(mesosHostName: String)
 
   def connect(mesosHostName: String): Future[(MesosConnection, Flow[SpecEvent, StateEvent, NotUsed])] = {
+    val composedLogger = log.withContext("mesosHostName", mesosHostName)
```

Comment:
Looking only at this usage example I don't quite understand: if the `composedLogger` already has context with `mesosHostName` - why do we need to implicitly pass it in the next line?

Reply:
(Discussed this offline) previously, a tuple was being created whenever `withContext` is being called, so the `log` was not actually being associated with a `context` but the abstractions helped to write `log.context`.

Decided that the tuple was not giving any added benefit, we changed from "composing a logger with context" to "composing a context" to be used with logger. Updated the example to reflect the same.